templates/includes/sidebar.html

```diff
@@ -1,4 +1,17 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated%}
+    <li>
+      User: <a href="{% url 'taxi:driver-detail' pk=user.id%}">{{user.username}}</a>
```

Comment:
Ensure that the `pk` parameter in the URL tag is correctly passed. If `user.id` is not the primary key for the driver, this could lead to incorrect URL generation.